Library/CSharpWrapper/src/NativeClassWrapper.cs

```diff
@@ -21,7 +21,7 @@ internal NativePointer(IntPtr pointer, byte ownsOwnData)
         public static NativePointer Zero => new NativePointer(IntPtr.Zero, 0);
     }
 
-    public class NativeClassWrapper
+    public class NativeClassWrapper : IDisposable
```

Comment:
The drawback here is that it makes all NativeClassWrapper-derived classes also IDisposable, which is incorrect for e.g. the *System singletons.